src/DIPS.Xamarin.UI/Internal/Xaml/FloatingActionMenu.xaml.cs

```diff
@@ -97,13 +97,16 @@ private async Task AnimateAll()
             var multiplier = 1;
             foreach (var menuButton in Children)
             {
-                var maxOpacity = menuButton.IsEnabled ? 1 : .5;
-                menuButton.TranslateTo(0, m_isExpanded ? 0 : -m_yTranslate * multiplier, 250, Easing.CubicInOut);
-                menuButton.Button.FadeTo(m_isExpanded ? 0 : maxOpacity, 250, Easing.CubicInOut);
-                menuButton.TitleFrame.FadeTo(m_isExpanded ? 0 : 1, 250, Easing.CubicInOut);
-                menuButton.BadgeFrame.FadeTo(m_isExpanded ? .5 : .95, 250, Easing.CubicInOut);
-
-                multiplier += 1;
+                if(menuButton.IsVisible)

```

Comment:
This won't work I think if the menu is expanded and then the button is set to invisible. Might be a case

Reply:
I agree on this, but I think we should create a issue for it and let this PR go through.